### error
Typing Error: This type cannot have a subtype.
 --> test/unit_tests/TypeCannotHaveSubtype.silly
21 | 
22 | def Something(a: int int) -> int:
                          ^^^ this type should not have another type in front of it
23 |    123;

Typing Error: There is no declaration with this name.
 --> test/unit_tests/TypeCannotHaveSubtype.silly
24 | 
25 | Something(123);
     ^^^^^^^^^ Something is undefined

Typing Error: Cannot call this expression of unknown type.
 --> test/unit_tests/TypeCannotHaveSubtype.silly
24 | 
25 | Something(123);
              ^^^^^ cannot call this expression of unknown type
###

def Something(a: int int) -> int:
	123;

Something(123);
